tests/ert/unit_tests/config/test_gen_kw_config.py

```diff
@@ -263,32 +262,32 @@ def test_gen_kw_is_log_or_not(
 )
 def test_gen_kw_distribution_errors(tmpdir, distribution, mean, std, error):
     with tmpdir.as_cwd():
-        config = dedent(
-            """
-        JOBNAME my_name%d
-        NUM_REALIZATIONS 1
-        GEN_KW KW_NAME template.txt kw.txt prior.txt
-        """
-        )
-        with open("config.ert", "w", encoding="utf-8") as fh:
-            fh.writelines(config)
+        template = "MY_KEYWORD <MY_KEYWORD>"
         with open("template.txt", "w", encoding="utf-8") as fh:
             fh.writelines("MY_KEYWORD <MY_KEYWORD>")
-        with open("prior.txt", "w", encoding="utf-8") as fh:
-            if distribution == "TRUNCATED_NORMAL":
-                fh.writelines(f"MY_KEYWORD {distribution} {mean} {std} -1 1")
-            else:
-                fh.writelines(f"MY_KEYWORD {distribution} {mean} {std}")
+
+        if distribution == "TRUNCATED_NORMAL":
+            distribution_line = f"MY_KEYWORD {distribution} {mean} {std} -1 1"
+        else:
+            distribution_line = f"MY_KEYWORD {distribution} {mean} {std}"
```

Comment:
This seems a bit excessive, but might be easier on the eye?

Reply:
Not sure what you mean, maybe you have an example alternative in mind?

Reply:
```
distribution_line = f"MY_KEYWORD {distribution} {mean} {std}"
if distribution == "TRUNCATED_NORMAL":
    distribution_line += " -1 1"
```

or

```
distribution_line = f"MY_KEYWORD {distribution} {mean} {std}{' -1 1' if distribution == 'TRUNCATED_NORMAL' else ''}"
```

Reply:
Aha, I'll just leave the original in place I think. The alternatives had a bit of stateful logic to them.